benchmark_v2/aggregate_benchmark.py

```diff
@@ -0,0 +1,41 @@
+import arkouda as ak
+import pytest
+
+
+def setup_agg(t="int"):
+    cfg = ak.get_config()
+    N = pytest.prob_size * cfg["numLocales"]
+
+    # Sort keys so that aggregations will not have to permute values
+    # We just want to measure aggregation time, not gather
+    keys = ak.sort(ak.randint(0, 2**32, N, seed=pytest.seed))
+    intvals = ak.randint(0, 2**16, N, seed=(pytest.seed + 1 if pytest.seed is not None else None))
+    g = ak.GroupBy(keys, assume_sorted=True)
+
+    if t == "int":
+        return g, intvals
+    else:
+        boolvals = (intvals % 2) == 0
+        return g, boolvals
+
+
+def run_agg(g, vals, op):
+    g.aggregate(vals, op)
+
+    return vals.size + vals.itemsize
+
+
+@pytest.mark.benchmark(group="GroupBy.aggregate")

```

Comment:
Just noting that we may reorganize how we do the grouping stuff. This isn't an issue to keep, but worth making sure we know it will most likely change.

Reply:
Brief discussion had on this - we're in agreement on grouping as necessary while acknowledging the grouping structure will likely change